packages/edge-functions/dev/node/main.ts

```diff
@@ -129,6 +129,32 @@ export class EdgeFunctionsHandler {
         return
       }
 
+      if (route.headers) {
+        const headerMatches = Object.entries(route.headers).every(([headerName, headerMatch]) => {
+          const requestHeaderValue = req.headers.get(headerName)?.split(', ').join(',')
+
+          if (headerMatch?.matcher === 'exists') {
+            return Boolean(requestHeaderValue)
+          }
+
+          if (headerMatch?.matcher === 'missing') {
+            return !requestHeaderValue
+          }
```

Comment:
Will this work as expected if header has empty string? I'm not sure how useful looking into such case is (do empty string header value happen in practice?), but:
```
Boolean((new Headers({ 'test': ''})).get('test').split(',').join(','))
```
would be `false`, so it would not pass `exists` check and would pass `missing` check, but I dunno if this would be a bug or not

Reply:
Fixed in https://github.com/netlify/primitives/pull/361/commits/b343ca19c8fb9629d113743cdf1c20daaafc12ec.